src/e2e-test/java/io/cdap/plugin/salesforcestreamingsource/stepsdesign/DesignTimeSteps.java

```diff
@@ -50,11 +73,59 @@ public void createNewLeadInSalesforce() throws UnsupportedEncodingException {
     lead.put("LastName", "LLname_" + uniqueId);
     lead.put("Company", uniqueId + ".com");
 
-    SalesforceClient.createLead(lead, "Lead");
+    SalesforceClient.createObject(lead, "Lead");
   }
 
   @Then("Enter unique Topic name as a Runtime argument value for key: {string}")
   public void fillUniqueTopicNameInRuntimeArguments(String runtimeArgumentKey) {
     SalesforcePropertiesPageActions.fillUniqueTopicNameInRuntimeArguments(runtimeArgumentKey);
   }
+
+  @Then("Update fields in salesforce")
+  public void updateFieldsInSalesforce() {
+    String uniqueRecordId = SalesforceClient.queryObjectId(SObjects.AUTOMATION_CUSTOM__C.value);
```

Comment:
Create a constant for SObjects.AUTOMATION_CUSTOM__C.value its getting reused in the complete class.